dio_cache/pubspec.yaml

```
@@ -10,7 +10,7 @@ environment:
 dependencies:
   meta: ^1.1.6
   logging: ^0.11.3
-  dio: ^2.1.7
+  dio: ^3.0.7
```

```suggestion
  dio: ^3.0.9
```

Latest version is now 3.0.9